extensions/src/main/java/cm/aptoide/pt/extensions/Randoms.kt

```diff
@@ -0,0 +1,27 @@
+package cm.aptoide.pt.extensions
+
+import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
+import java.util.Locale
+import kotlin.random.Random
+import kotlin.random.nextInt
+
+fun getRandomString(
```

Comment:
I'm glad you also came up with the idea to extract this to `extensions` :-)
Though I already extracted this in my PR. Lets merge that first and then rebase this PR on top.